README.md

```diff
@@ -23,19 +23,30 @@ EntandoDeBundles served in the cluster can be installed in an EntandoApp using t
-| ENTANDO_URL                                                   | The URL to access the Entando App instance.                                                                   |
-| ENTANDO_APP_NAMESPACE                                         | The kubernetes namespace where the entando app is running. Default to `test-namespace`;                       |
-| ENTANDO_APP_NAME                                              | The entando app name that this service is in. Defaults to `test-entando`.                                     |
-| ENTANDO_BUNDLE_TYPE                                           | The bundle type that should be handled by this service. It can be `git` or `npm`. The default value is `git`. |
+| Group | Name | Value [default] | Description |
+| :---   | :--- | :--- |:--- |
+| General | ENTANDO_URL                 |  | The URL to access the Entando App instance                            |
+|     | ENTANDO_APP_NAMESPACE                                        | [test-namespace] | The kubernetes namespace where the entando app is running |
+|     | ENTANDO_APP_NAME                                             | [test-entando] | The entando app name that this service is in |
+|    | ENTANDO_ECR_DEAPP_REQUEST_RETRIES    | [3] |  Number of times the CM retries the component create/update before giving up     |
+|    | ENTANDO_ECR_DEAPP_REQUEST_BACKOFF       | [5] |  Seconds to wait before the next attempt is executed     |
+|    | ENTANDO_ECR_POSTINIT        |  |  Configuration of the postinit process      |
+|    | ENTANDO_CONTAINER_REGISTRY_CREDENTIALS   | [null] | Credentials for each container registry    |
+|    | MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE     | [not enabled] |  To enable component-manager /actuator/info     |
+|Spring Security| SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_OIDC_ISSUER-URI  | |  The issuer of the token, e.g., http://insecure-keycloak-cacms.apps.serv.run/auth/realms/entando                 |
+|     |SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT-ID   |  | The client id for the service                                                                                 |
+|     |SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT-SECRET| | The client secret         |
+| Database | DB_VENDOR             | [postgres] | Which database will be used     |
+|    | DB_HOST      | [localhost] | Database host |
+|    | DB_PORT  | [5432] | Database port |
+|    | DB_NAME    | [digital_exchange] | Database name |
+|    | DB_OPTIONS           | [useSSL=false] | Database options |
+|    | DB_USER       | [admin] | Database user |
+|    | DB_PASS        | [admin] | Database password |
+| Bundle Related | ENTANDO_BUNDLE_TAGS_TYPES       | dev, [prod] | To generate EntandoDeBundle CRs using tags to select for dev, prod, or both |
+|     | ENTANDO_BUNDLE_TYPE           | npm, [git] | The bundle type that should be handled by this service |
```

Comment:
1. We removed npm support a long time ago. I think we should remove this from the list. 
2. Is this var even used now?  Even the git type is deprecated now so I'm not sure what this one does anymore.

Reply:
removed from list